Team8923_PowerPlay/src/main/java/org/firstinspires/ftc/team8923_PowerPlay/BaseTeleOp.java

```diff
@@ -49,17 +50,18 @@ public void driveMechanism() {
         }
     }
 
-    /**
-     * press the X button, claw opens
-     * press the A button, claw closes
-     */
+
+     // press the X button, claw opens
+     // press the A button, claw closes
+
```

Comment:
Please don't remove the javadoc comments. That's how you should be documenting all methods.
```suggestion
    /**
     * press the X button, claw opens
     * press the A button, claw closes
     */
```